warp10/src/main/java/io/warp10/script/functions/LASTSORT.java

```diff
@@ -36,7 +36,7 @@
 
   private static final Comparator<GeoTimeSerie> LAST_COMPARATOR = new Comparator<GeoTimeSerie>() {
     
-    private final Comparator<Metadata> INNER_COMP = new MetadataTextComparator(null);
+    private final Comparator<Metadata> INNER_COMP = new MetadataTextComparator(null, false);
```

Comment:
Remove `false` as it seems to indicate this sort does not take into account attributes, which is not the case (see previous comment).

Reply:
done